src/Library/Ui/Premiums.php

```diff
@@ -0,0 +1,118 @@
+<?php //phpcs:ignore WordPress.Files.FileName
```

Comment:
For maintainability, would it be easier just to update the existing class?

For example, instead of adding this:
```
if ( \Boldgrid\Library\Library\NoticeCounts::is_unread( 'boldgrid-backup-premium-features', $this->id ) ) {
			$markup .= '<div class="card-ribbon"><span>NEW</span></div>';
}
```

We could add:
```
if ( ! empty( $this->ribbon ) ) {
     $markup .= '<div class="card-ribbon"><span>' . $this->ribbon . '</span></div>';
}
```